src/index.js

```diff
@@ -220,6 +221,13 @@ function App() {
           component={MenteeAddReview}
         />
 
+        <PrivateRoute
```

Comment:
is this working? I know we just merged a PR that completely refactored the auth portion of our app.